ui/src/app/store/zone/constants.ts

```diff
@@ -0,0 +1,26 @@
+/* The following constants should be made generic, usable across models. */
+export const ACTION_STATUS = {
+  failed: "failed",
+  processing: "processing",
+  successful: "successful",
+} as const;
+/***************************/
+
+export const ZONE_PK = "id";
+
+export const ZONE_MODEL = "zone";
```

Comment:
I've replaced the `ZoneMeta` enum with these two constants. I'm not married to this change, but my general preference is not to use enums since I feel like they trip me up by being both types and variables.

Reply:
Reverted to using the `ZoneMeta` enum